hugr-passes/Cargo.toml

```diff
@@ -14,6 +14,10 @@ categories = ["compilers"]
 
 [dependencies]
 hugr-core = { path = "../hugr-core", version = "0.10.0" }
+portgraph = { workspace = true }
+# This ascent commit has a fix for unsoundness in release/tag 0.6.0:
+ascent = {git = "https://github.com/s-arash/ascent", rev="9805d02cb830b6e66abcd4d48836a14cd98366f3"}
```

Comment:
Merging this will prevent us from publishing to crates.

Ascent 0.7.0 is now out, so please fix before merging.

Reply:
Thank you! :-) and done